src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let carCost = 0;
+  const costForDays = days * 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const mediumTerm = 3;
+  const mediumTermDiscount = 20;
+
+  if (days >= 1 && days <= 2) {
+    carCost = costForDays;
+  } else if (days >= mediumTerm && days <= 6) {
+    carCost = costForDays - mediumTermDiscount;
+  } else if (days >= longTerm) {
+    carCost = costForDays - longTermDiscount;
+  }
```

Comment:
The conditions can be simplified by checking for the `longTerm` condition first, then the `mediumTerm` condition, and finally returning `costForDays` if none of the conditions are met. This will make the code more readable and efficient as per checklist item 4 and 5.